### DISCUSSION TITLE
OSU does not register mouse or keyboard* input in Samsung Dex

### DISCUSSION THREAD

**Discussion by EDLLT** [ppy/osu#26475] [Q&A] [answered]
### Type

Game behaviour

### Bug description

When opening OSU inside Samsung Dex, all the mouse input is discarded as well as the keyboard

Keys that work consistently from my observation are Enter and Escape, the rest do get detected(during the initial bootup where the OSU circle appears) however they don't interact with the UI whatsoever

Some keyboard shortcuts like o(for settings), Enter, p(basically enter), escape, the function keys do work

For example, pressing down the UP, LEFT, RIGHT, DOWN arrow keys get registered during the initial boot of the game(in which the OSU circle appears allowing you to proceed to the menu). However, trying to navigate the UIs once you reach said menu using the arrow keys does not work.

Mouse however, does NOT register at all anywhere anytime

I then tried to switch to mirroring mode in Dex(Basically mirrors the phone screen) then the mouse and keyboard started behaving normally.

Phone: G781B S20 FE 5G
Android: 12
One UI: 4.1
### Screenshots or videos

![Uploading Screenshot_20240111_101623.jpg…]()

### Version

Latest version 2023.1231.0

### Logs

NOTE: I can't seem to export logs while in Dex mode as the arrow keys and mouse aren't registering any keys pressed
(Tried using input tap X Y -d DISPLAY_ID, Automate by llamalab, key mapper, BobbleKeyboard/android_touch to simulate touch input)

I could turn off dex then use OSU outside of Dex mode, would that log be appropriate to provide?

**Comment by EDLLT** [comment]
Why was this converted into a conversation? It's a bug

**Comment by EDLLT** [comment]
Just noticed someone created the same issue which then got converted into a conversation
https://github.com/ppy/osu/discussions/25138

**Comment by Joehuu** [comment, marked as answer]
Duplicate of https://github.com/ppy/osu/discussions/25138
